lib/ipc_int.h

```diff
@@ -92,6 +92,7 @@ struct qb_ipcc_connection {
 	char name[NAME_MAX];
 	int32_t needs_sock_for_poll;
 	gid_t egid;
+	uid_t euid;
```

Comment:
I'd add this to the end of the struct so you don't have to bump the shared library version. Uglier, but library bumps are a hassle